README.md

```diff
@@ -1 +1,44 @@
-# xblock-skytap
\ No newline at end of file
+# xblock-skytap
+
+## What is it?
+
+An XBlock for integrating [Open edX](https://open.edx.org/) and [Skytap](https://www.skytap.com/).
+
+## Support
+
+| Open edX Release | Tag |
+|:-----------------|:---:|
+| Juniper | v0.2 |
+| Ironwood | v0.1 |
+
+## Install 
+
+See [edX Installing an XBlock](http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/install_xblock.html?highlight=install%20xblock)
+for details on how to install this XBlock on your Open edX instance.
+
+```python
+pip install "git+https://github.com/open-craft/xblock-skytap@master#egg=xblock-skytap"
+```
+
+## Testing
+
+The test suite uses `tox`, so install it into a virtualenv to run the tests:
+
+```bash
+pip install tox
+```
+
+Accordingly, all you need to do in the repo's directory is to run the following:
+
+```bash
+tox
+```
+
+## Installing on DevStack
+
+1. Start your *edX devstack*
+2. Once the *edX devstack* is up, run `make studio-shell`
+3. `source ../venvs/edxapp/bin/activate`
```

Comment:
Step 3 isn't required, you're already in the venv when you enter the shell:

```bash
devstack$ make studio-shell
docker exec -it edx.devstack.studio env TERM=xterm-256color /edx/app/edxapp/devstack.sh open
root@studio:/edx/app/edxapp/edx-platform# which python
/edx/app/edxapp/venvs/edxapp/bin/python
```

Reply:
Oh, I never paid attention to that, thank you :+1: